value_test.go

```diff
@@ -430,11 +430,11 @@ func TestValueGC4(t *testing.T) {
 func TestPersistLFDiscardStats(t *testing.T) {
 	// TODO(ibrahim): This test is failing because compactions are not
 	// happening and so no discard stats are generated.
-	t.Skip()
 	dir, err := ioutil.TempDir("", "badger-test")
 	require.NoError(t, err)
 	defer removeDir(dir)
 	opt := getTestOptions(dir)
+	opt.NumLevelZeroTables = 1
```

Comment:
Please add a comment in code for why we need to do this

Reply:
Done